pysollib/games/harp.py

```diff
@@ -333,6 +337,69 @@ def startGame(self):
     getQuickPlayScore = Game._getSpiderQuickPlayScore
 
 
+# ************************************************************************
+# * Churchill Solitaire
+# ************************************************************************
+# https://boardgames.stackexchange.com/questions/29254/rules-for-churchill-solitaire
+
+class Churchill_DevilStack(OpenStack):
+    pass
+
+class Churchill_RowStack(KingAC_RowStack):
```

Comment:
Please add a second line break between each of these classes (see https://www.flake8rules.com/rules/E302.html)